scripts/build_osm_network.py

```diff
@@ -908,12 +911,33 @@ def built_network(
 
     sets_path_to_root("pypsa-earth")
```

Comment:
This is not related to this PR, but can we drop this? [it can be a separate good first issue]
We should avoid using sets_path_to_root; it leads to problems when users rename the folder name

Reply:
Happy to do :) But agree that it deserves a dedicated PR

Reply:
The issue is tracked in #639, have updated the tags